system-int-test/test/cmr/system_int_test/search/granule_orbit_search_test.clj

```diff
@@ -81,6 +81,22 @@
         "CMR-4722: Search crossing the equator should not erroneously find the granule"
         [] [-128.32 53.602 -46.758 -1.241]))))
 
+(deftest orbit-bug-CMR-6431
+  (let [coll (d/ingest-concept-with-metadata-file "CMR-6431/sample-orbit-collection.xml"
+                                                  {:provider-id "PROV1"
+                                                   :concept-type :collection
+                                                   :native-id "orbit-collection"
+                                                   :format-key :dif10})]
+    (index/wait-until-indexed)
+
+    (testing "Orbit search will not throw exception."
```

Comment:
can you add "when crossing range is nil" to this to explain what is being tested a little better?

Reply:
Done.